src/home-page/home-page.component.ts

```diff
@@ -9,27 +10,65 @@ import {Api} from '../environments/environment';
 })
 export class HomePageComponent implements OnInit {
   public cityName: string;
-  public cities = [];
+  public cities: [] = [];
```

Comment:
array of what?
It's typescript, we should declare every variable type, same goes for lines 14, 15, 16 below 🕺